specification/09-coveio_abi.adoc

```diff
@@ -105,3 +127,17 @@ struct sbiret sbi_covg_map_interface_mmio(unsigned long device_if_id
                                           unsigned long hpa_offset,
                                           unsigned long size);
 ----
+
+[#sbi_covg_run_interface()]
+===== Run Device Interface (FID TBD)
```

Comment:
to match `sbi_covg_stop_interface`, maybe we can call it `sbi_covg_start_interface` ?

Reply:
I did a global search, still find `sbi_covg_run_interface` ...

Reply:
Ah, sorry. That *should* be fixed now...